pull request livecell

xjzhaang opened the pull request.

xjzhaang committed:
added livecell

sbinnee commented:
1. I love `tqdm` too. But I don't think you need it. And you forgot to import it? I would say remove it. 
2. I prefer `pathlib` over `os.path`.
3. It's safe to set `save_tif` to False by default, because I don't want users accidentally write new files. I am thinking of putting warning to notify users about the option.

xjzhaang committed:
1. removed tdqm, replaced os
2. set save_tif to false as default
3. added warning message if tif masks do not exist

sbinnee committed:
Merge branch 'main' into livecell
